config/install/README.md

```diff
@@ -0,0 +1,148 @@
+# Install Kuadrant and Sail via OLM
+
+## Prerequisites  
+- Clone the[ Kuadrant-operator](https://github.com/Kuadrant/kuadrant-operator) repo
+- OLM (operator lifecycle manager)
+- cert-manager 
+- - [cert-manager Operator for Red Hat OpenShift](https://docs.openshift.com/container-platform/4.16/security/cert_manager_operator/cert-manager-operator-install.html)
```

Comment:
Sorry i got the markdown wrong for this its not two dash for indented bullet point you actually just indent the one dash